### DISCUSSION TITLE
URL of EPA Data Commons

### DISCUSSION THREAD

**Discussion by MoLi7** [USEPA/flowsa#336] [Q&A] [answered]
Hi team, I was directed to (the same?) Data Commons by two URLs:

1. https://**dmap**-data-commons-ord.s3.amazonaws.com/index.html
2. https://**edap**-data-commons-ord.s3.amazonaws.com/index.html

It seems some FBAs, e.g. EPA_GHGI_xxx_2020, are only available in the first one but not the second. Should I refer to the first for the most up-to-date data?

Thanks!

**Comment by bl-young** [comment, marked as answer]
Recently EPA moved the site hosting data commons to the first link (`dmap`). See this PR in esupy: https://github.com/USEPA/esupy/pull/45

All packages should now pull through that. I don't think we've yet implemented in useeior however.

The old site is still active for now, but all datasets should have been moved over. Any new ones in the last few months, including GHGI_2020 would only be aviailable on `dmap`.
